File: app/src/main/java/com/kml/internalRoomDatabase/GameDao.java
package com.kml.internalRoomDatabase;

import androidx.lifecycle.LiveData;
import androidx.room.Dao;
import androidx.room.Query;


import java.util.List;

@Dao
public interface GameDao
{
    @Query("SELECT * FROM gameTable")
    LiveData<List<Game>> getAllGames();
}

File: app/src/main/java/com/kml/data/database/GameDao.kt
package com.kml.data.database

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Query
import com.kml.models.entitiy.Game

@Dao
interface GameDao {
    @get:Query("SELECT * FROM gameTable")
    val allGames: LiveData<List<Game>>
}